components/ingest-service/grpc/grpc.go

```diff
@@ -32,6 +33,8 @@ import (
 //
 // Maybe even spawn multiple servers
 func Spawn(opts *serveropts.Opts) error {
+	var client backend.Client
```

Comment:
curiour: What is this for? 🤔

Reply:
I put this here to have the client be a `backend.Client` and not a `elastic.Backend`.